include/grpc/grpc_security.h

```diff
@@ -698,6 +698,64 @@ GRPCAPI grpc_channel_credentials* grpc_local_credentials_create(
 GRPCAPI grpc_server_credentials* grpc_local_server_credentials_create(
     grpc_local_connect_type type);
 
+/** --- S2A channel/server credentials --- **/
+
+namespace experimental {
+
+/** The main interface for the S2A credentials options. The options will contain
+ *  information that will be passed from gRPC to the TSI layer, such an ordered
+ *  list of supported ciphersuites or target service accounts. The S2A client
```

Comment:
Will S2A also use target service accounts for strawman server authorization check? I think that feature is specific to ALTS.